.travis.yml

```diff
@@ -44,3 +44,20 @@ script:
 
 after_success:
     - php vendor/bin/coveralls -v
+
+before_deploy:
+    - 'sed -i "s/^{/\{\n    \"config\": {\"platform\": {\"php\": \"5.3.6\"}},/" composer.json'
+    - curl -LSs http://box-project.github.io/box2/installer.php | php
+    - composer update --no-dev --no-interaction --optimize-autoloader --prefer-stable
+    - php -d phar.readonly=false box.phar build
+
+deploy:
+    provider: releases
```

Comment:
Releases in Github are build on top of tags.
Deployments on Travis are build on top of anything. So we say to trigger it only for tags.